definitions/npm/commander_v2.x.x/flow_v0.28.x-/commander_v2.x.x.js

```diff
@@ -243,7 +248,7 @@ declare module "commander" {
      *
      * @api public
      */
-    outputHelp(): void;
+    outputHelp((defaultHelp: string) => string): void;
```

Comment:
based on docs, this is optional, you can use e.g `outputHelp(cb?: ?(defaultHelp: string) => string): void;`

Reply:
OK, I will fix this.